public/manifest.json

```diff
@@ -1,20 +1,12 @@
 {
     "author": "BlockWallet",
     "background": {
-        "scripts": [
```

Comment:
this lint change will cause the pipeline to fail. Please run `yarn run lint:fix` standing in the UI folder to fix it